great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -2118,6 +2118,8 @@ def _resolve_add_checkpoint_args(  # noqa: PLR0913
                 "Must either pass in an existing checkpoint or individual constructor arguments (but not both)"
             )
 
+        action_list = action_list or self._determine_default_action_list()
```

Comment:
I'd love to push this down into the `Checkpoint` but that would require an `isinstance(context, CloudDataContext)`. Thoughts? Feels a little bad here.

Reply:
Perhaps another selling point for `CloudCheckpoint`?